tox.ini

```diff
@@ -1,20 +1,40 @@
 [tox]
-envlist = pre-commit,py{27,35,36,37}-sql{11,12,13}
+envlist = pre-commit,py{27,35,36,37,38,39}-sql{12,13},flake8
 skipsdist = true
 minversion = 3.7.0
 
+[gh-actions]
+python =
+    2.7: py27
+    3.5: py35
```

Comment:
These can probably be removed since they're not being tested against.

Reply:
Will follow up with coverage fix and removing these tests